include/manager.h

```diff
@@ -0,0 +1,69 @@
+#ifndef _MANAGER_H
+#define _MANAGER_H
+
+#include <assert.h>
+#include <string.h>
+#include <unistd.h>
+#include <dlfcn.h>
+#include <sys/auxv.h>
+
+// vDSO wrapper needed includes
+#include <sys/time.h>
+//#include <sys/vdso.h>
```

Comment:
I guess we can kill this include now?

Reply:
It might be worth revisiting whether we want to do a finer redirect with the vDSO functions at some point, but for now I've removed it in https://github.com/capablevms/CHERI-ELF-comp/pull/1/commits/26914cd1d724003393bc24b2f5bb54a2899692c2.